pyproject.toml

```diff
@@ -64,3 +64,9 @@ filterwarnings = [
     [[tool.towncrier.section]]
         name = ""
         path = ""
+
+[tool.ruff]
```

Comment:
Ruff specifies that compatibility with Black requires setting the same line length for both, so it's probably best to set it explicitly for ruff as well as for Black.